flake8_import_order/checker.py

```diff
@@ -58,7 +58,7 @@ def check_order(self):
         elif style_option == 'smarkets':
             style = Smarkets(imports)
         else:
-            raise AssertionError("Unknown style {)".format(style_option))
+            raise AssertionError("Unknown style {}".format(style_option))
```

Comment:
This is tangential but I noticed it and thought I'd tuck it into this PR. I can pull it into a separate PR if you want @pgjones